tools/export/coide/arch_max.coproj.tmpl

```diff
@@ -6,7 +6,7 @@
       <Compile>
         <Option name="OptimizationLevel" value="4"/>
         <Option name="UseFPU" value="0"/>
-        <Option name="UserEditCompiler" value="-fno-common; -fmessage-length=0; -Wall; -fno-strict-aliasing; -fno-rtti; -fno-exceptions; -ffunction-sections; -fdata-sections; -std=gnu++98"/>
+        <Option name="UserEditCompiler" value="-fno-common; -fmessage-length=0; -Wall; -fno-strict-aliasing; -fno-rtti; -fno-exceptions; -ffunction-sections; -fdata-sections; -std=gnu++14"/>
```

Comment:
Question: How do we differentiate Armc5 / Armc6 profiles in exporters?

Reply:
I don't know, but not sure I understand the question. This template must be for GCC or ARMC6, as `-std` is an option only for them.

There may be further exporter changes required for toolchains where the standard is specified via an XML option, which may not be visible on my search.

Reply:
My bad, I thought this was default template for uVision didn;t see `coide`

Reply:
Deepika's right though, we'll need to update the uvision template a bit more to play nicely with the C++ and C version. The language versions are selected by the IDE GUI.